devel/libblocksruntime/Portfile

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                libblocksruntime
+version             0.4.1
+categories          devel
+platforms           darwin
+license             MIT
+maintainers         gmail.org:adsun701
+description         compiler-rt Blocks runtime library for Clang
+long_description    libblocksruntime is a target-independent implementation of Apple "Blocks" runtime interfaces.
+homepage            https://compiler-rt.llvm.org
+master_sites        http://ftp.debian.org/debian/pool/main/libb/libblocksruntime/
+distfiles           ${name}_${version}.orig.tar.gz
```

Comment:
`distfiles` should not normally be set directly. Just set the `distname`.

Reply:
I set the distname, but distfiles still needs to be set due to an underscore between the name and version of the source tarball instead of a dash.

Reply:
Setting the `distname` to the wrong value is not useful. By default, `distfiles` is `${distname}${extract.suffix}`. If the inside folder is a different name, then the worksrcdir needs to be set. So I think the following should work:

```
distname            ${name}_${version}.orig   
worksrcdir          ${name}-${version}
```

Reply:
I removed the distfiles, corrected distname, and added the worksrcdir. Thanks for the feedback!